webrender/src/frame_builder.rs

```diff
@@ -616,17 +616,23 @@ impl FrameBuilder {
         &mut self,
         clip_and_scroll: ClipAndScrollInfo,
         info: &LayerPrimitiveInfo,
-        color: &ColorF,
+        operation: &FillOrClear,
         flags: PrimitiveFlags,
     ) {
-        let prim = RectanglePrimitive { color: *color };
-
-        // Don't add transparent rectangles to the draw list, but do consider them for hit
-        // testing. This allows specifying invisible hit testing areas.
-        if color.a == 0.0 {
-            self.add_primitive_to_hit_testing_list(info, clip_and_scroll);
-            return;
-        }
+        let prim = match operation {
+            &FillOrClear::Fill(ref color) => {
```

Comment:
Could also be `RectangleContent::Fill/Clear`

Reply:
I went with this suggestion, `RectangleContent::Fill/Clear`. Thanks!